amazon-ec2/2.3/modules/ROOT/pages/amazon-ec2-connector-examples.adoc

```diff
@@ -79,19 +85,21 @@ The corresponding XML configuration should be as follows:
 . If the connection is successful, click *OK* to save the configurations.
 +
 Otherwise, review or correct any incorrect parameters, then test again.
-. In the Properties editor, configure the remaining parameters:
-+
-[%header,cols="30s,70a"]
-|===
-|Studio Field |Value
-|Display Name |Create Volume (or any other name you prefer)
-2+|Basic Settings
-|Extension Configuration |Amazon_EC2_Configuration (the name you created for the global element)
-2+|General
-|Availability Zone |`us-east-1a` (or any other availability zone to which you have access)
-|Size |5 (the size of the volume, in gibibytes GiBs)
-|Volume Type |Standard (the default volume type)
-|===
+
+== Configure Additional Properties
+
+. Select the *Create volume* operation on the Studio canvas.
+. In the *Create volume* *General* tab, enter the following values:
```

Comment:
The formatting symbols here make clear that the tab name is *General* and it is part of the *Create volume* operation display...but when transformed, this is going to look like the name of the tab is "Create volume General".
```suggestion
. In the *General* tab, enter the following values:
```